fastlane/lanes/localization.rb

```diff
@@ -321,16 +327,22 @@
     # (like `FileUtils` calls here) run relative to the `./fastlane` folder, but the `*_DIR_PATH` we use are relative to the repo root.
     # See: https://docs.fastlane.tools/advanced/fastlane/#directory-behavior
     Dir.chdir('..') do
-      FileUtils.mkdir_p(FROZEN_STRINGS_DIR_PATH)
-      FileUtils.cp(MAIN_STRINGS_PATH, FROZEN_STRINGS_DIR_PATH)
+      FileUtils.mkdir_p(WORDPRESS_FROZEN_STRINGS_DIR_PATH)
+      FileUtils.cp(WORDPRESS_MAIN_STRINGS_PATH, WORDPRESS_FROZEN_STRINGS_DIR_PATH)
+      FileUtils.mkdir_p(JETPACK_FROZEN_STRINGS_DIR_PATH)
+      FileUtils.cp(JETPACK_MAIN_STRINGS_PATH, JETPACK_FROZEN_STRINGS_DIR_PATH)
     end
-    git_commit(path: File.join(FROZEN_STRINGS_DIR_PATH, 'strings.xml'), message: 'Freeze strings for translation', allow_nothing_to_commit: true)
+    git_commit(
+      path: [File.join(WORDPRESS_FROZEN_STRINGS_DIR_PATH, 'strings.xml'), File.join(JETPACK_FROZEN_STRINGS_DIR_PATH, 'strings.xml')],
+      message: 'Freeze strings for translation',
+      allow_nothing_to_commit: true
+    )
```

Comment:
Have you considered doing two commits? Would it be useful to be able to look at the diff for only one app at a time?

Reply:
I thought about it (in fact that's what I did with my initial implementation, splitting this lane in two sublanes, and having the current lane calling both)… but considered it wasn't really worth it as (1) it created more noise both in the Fastfile code and in the commit list (2) since the files modified by each case are separate, having 2 commits each containing one file would not benefit us much as opposed to one commit with two distinct files, because they changes on each of the two files are already independant, so if we want to look at the diff for only one app at a time we already can, by just looking at the corresponding `.xml` file.

Reply:
👍